README.md

```diff
@@ -1,2 +1,11 @@
 # `stack` CLI
 Airbus UTM Platform `stack` CLI
+
+## Installation
+
+### macOS/Linux
```

Comment:
Would be more obvious what's happening with we had a separate macOS and linux installation command.

Reply:
I haven't tested on Catalina but it should return `darwin`. I have deliberately made it a one liner to add it easily within pipelines. I have also added OS specific commands for clarity.